src/test/java/io/rtr/conduit/util/LoggingAmqpCallbackHandler.java

```diff
@@ -0,0 +1,35 @@
+package io.rtr.conduit.util;
+
+import com.rabbitmq.client.ShutdownSignalException;
+import io.rtr.conduit.amqp.AMQPConsumerCallback;
+import io.rtr.conduit.amqp.AMQPMessageBundle;
+import io.rtr.conduit.amqp.ActionResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.charset.StandardCharsets;
+
+public class LoggingAmqpCallbackHandler implements AMQPConsumerCallback {
```

Comment:
That kinda voids https://renttherunway.jira.com/browse/DEVELOPERS-123 (Standard implementation for AMQP Callbacks) partially, thank you! 👍